### ISSUE TITLE
transaction without change address

### ISSUE THREAD

**Issue by parin13** [blockcypher/explorer#324]
I did a transaction using blockcypher api and here is the [reference](https://live.blockcypher.com/btc-testnet/tx/e60a726a870f8c0895dd6598da683f982cdacb36f4a8469a8b670f8804155030/) for the same, 
the object I used for creating the transaction is mentioned below where I have not mentioned any change address, still all the change is returned to the original address, how is it possible? Should we not mention the change address explicitly?
What is utxo of `mwbiDnK1tQfhz9vTz6UwkSH9NmD3Ho4uM6` in which its new balance was updated  ? 
 ```
{
            "inputs":[
                {
                    "addresses":[
                        mwbiDnK1tQfhz9vTz6UwkSH9NmD3Ho4uM6
                    ]
                }
            ],
            "outputs":[
                {
                    "addresses":[
                        moC17MzbvgUJAZWEqoRiJJ8S7YdxyeXoVL
                    ],
                    "value": 00002)
                }
            ],
        "preference": "low"
        }
```

**Comment by kdove527** [comment]
Cash$0.009876543210-$,.9876543210-$,.9876543210-$,.9876543210-$,.9876543210-$,.9876543210-$,.ACH Account Number 40902624064770Routing Number 041215663Hide Account Details
Recent History
Show More

Amazon.com14h
-$70.35
DeclinedStatusDeclinedTimeNov 20, 2020TypeOnlineDescriptionAMZN Mktp US Amzn.com/bill, WA

Amazon.com14h
-$161.18
DeclinedStatusDeclinedTimeNov 20, 2020TypeOnlineDescriptionAMZN Mktp US Amzn.com/bill, WA

Amazon.com14h
-$78.83
DeclinedStatusDeclinedTimeNov 20, 2020TypeOnlineDescriptionAMZN Mktp US Amzn.com/bill, WA

Amazon.com14h
-$6.39
DeclinedStatusDeclinedTimeNov 20, 2020TypeOnlineDescriptionAMZN Mktp US Amzn.com/bill, WA

Amazon.com14h
-$431.92
DeclinedStatusDeclinedTimeNov 20, 2020TypeOnlineDescriptionAMZN Mktp US Amzn.com/bill, WA
Interest Earned
So far this month$0.00Next pay dateNov 30, 2020APY0.30%*Cash earning interest$0.00
Debit Card
9199Show Card NumberOnOffLock CardOnOffLock WithdrawalsOnOffChange PINReport a Problem
Pay Bills & Direct Deposit
ACH Account Number••••••••••4770Routing Number•••••5663ShowYour routing number is provided by Sutton Bank. Wire transfers are not supported through your routing and ACH account numbers.*The Annual Percentage Yield (APY) paid by program banks is 0.30% as of March 16, 2020. APY might change at any time at the program banks’ discretion. Neither Robinhood Financial LLC nor any of its affiliates are banks.Free StocksPortfolioCashMessagesAccount       

    On ‎Saturday‎, ‎November‎ ‎21‎, ‎2020‎ ‎12‎:‎40‎:‎22‎ ‎PM‎ ‎CST, CryptoKtm <notifications@github.com> wrote:  
 
 


I did a transaction using blockcypher api and here is the reference for the same,
the object I used for creating the transaction is mentioned below where I have not mentioned any change address, still all the change is returned to the original address, how is it possible? Should we not mention the change address explicitly?
What is utxo of mwbiDnK1tQfhz9vTz6UwkSH9NmD3Ho4uM6 in which its new balance was updated ?
{
           "inputs":[
               {
                   "addresses":[
                       mwbiDnK1tQfhz9vTz6UwkSH9NmD3Ho4uM6
                   ]
               }
           ],
           "outputs":[
               {
                   "addresses":[
                       moC17MzbvgUJAZWEqoRiJJ8S7YdxyeXoVL
                   ],
                   "value": 00002)
               }
           ],
       "preference": "low"
       }
